lib/models/spaces/space-models.ts

```diff
@@ -14,20 +14,23 @@ export namespace SpaceModels {
     export interface IAddSpaceData {
         name: string;
         codename?: string;
+        webSpotlightRootItem: SharedModels.ReferenceObject
     }
 
     export class Space implements SharedModels.IBaseModel<SpaceContracts.ISpaceContract> {
 
         public id!: string;
         public name!: string;
         public codename!: string;
+        webSpotlightRootItem!: SharedModels.ReferenceObject
```

Comment:
Shouldn't there be a 'public' identifier?

Reply:
I didn't intend to include these changes in these branch, my mistake. I'll fix this when I submit the root item update later this week.

Reply:
Fixed